### PR TITLE
Delete equipment from map and SLD

### PR DESCRIPTION
**Pull request by archendel**

### CODE REVIEW

**Review by laurgarn — APPROVED** [review]
Tests: OK
Code: OK
Console: OK

**Review by Meklo — APPROVED** [review]

**Review by dbraquart — CHANGES_REQUESTED** [review]
code review: pending
tests:
- KO : delete item is displayed twice for the lines in the map, and for lines/2WT in the SLD
- question / KO: can we delete a HVDC conversion station ? If we do so, we've got an error message

**Review by dbraquart — APPROVED** [review]
tests: all good
review: minor remarks about naming

File: src/components/menus/base-equipment-menu.js [hunk 1]
```diff
@@ -58,10 +60,42 @@ const ItemViewInSpreadsheet = ({
     );
 };
 
+const ItemDeleteEquipment = ({
```
[the remark this answers is not in the document]
Comment (archendel) [reply src/components/menus/base-equipment-menu.js]: done for both

File: src/components/menus/base-equipment-menu.js [hunk 2]
```diff
@@ -78,20 +112,50 @@ const BaseEquipmentMenu = ({
     }
     const equipment = getEquipment(equipmentType, equipmentId);
     const { getNameOrId } = useNameOrId();
+    const equipmentsWithBranch = [
```
[the remark this answers is not in the document]
Comment (archendel) [reply src/components/menus/base-equipment-menu.js:141]: done

File: src/components/menus/base-equipment-menu.js [hunk 3]
```diff
@@ -78,20 +112,50 @@ const BaseEquipmentMenu = ({
...
+        equipments.lines,
+        equipments.twoWindingsTransformers,
+    ];
+    const equipmentsNotDeletable = [
```
[the remark this answers is not in the document]
Comment (archendel) [reply src/components/menus/base-equipment-menu.js:145]: done

**Pull request merged**